third-party/realdds/include/realdds/dds-device-impl.h

```diff
@@ -60,12 +107,10 @@ class dds_device::impl
 
     bool _running = false;
 
-    size_t _num_of_profiles = 0;
-    std::vector< topics::device::notification::video_stream_profile > _video_profiles;
-    std::vector< topics::device::notification::motion_stream_profile > _motion_profiles;
-    std::unordered_map< size_t, std::string > _profile_to_profile_group;
-    std::unordered_map< std::string, std::vector< size_t > > _video_profile_indexes_in_profile_group;
-    std::unordered_map< std::string, std::vector< size_t > > _motion_profile_indexes_in_profile_group;
+    size_t _expected_num_of_streams = 0;
+    size_t _received_num_of_streams = 0;
+    std::map< size_t, std::shared_ptr< dds_stream > > _streams;
```

Comment:
Not clear what the `size_t` refers to

Reply:
Referred to stream ID.
But this is not enough because some cameras (D455) are sending Infrared 1 and Infrared 2 as same UID with different index. Now map key is <uid, index> pair. And added a comment to explain